dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServletContainerInitializerFilterServletNameMappingTest.java

```diff
@@ -34,7 +34,7 @@
  * 1. Test the dynamic add servlet after the dynamic filter, which adds a servlet-filter name mapping for that servlet, to make sure no NPE.
  * Also verify that the filter mapping flow is not breaking (i.e continue to the next filter mapping during filter chain build up)
  *
- * 2. Verify the new WARNING message:
+ * 2. If (1) successes, verify the new WARNING message:
```

Comment:
Can we change "successes" to "If (1) is successful"?

Reply:
done